public/javascripts/timeline.js

```diff
@@ -145,6 +396,14 @@ function hour2Date(hour) {
     return now;
 }
 
```

Comment:
Consider adding a second, optional argument to hour2Date that takes a minute. If you do this, you can eliminate the minute2Date function, which does almost the exact same thing.